internal/processor/otel/traces.go

```diff
@@ -345,13 +346,13 @@ func TranslateTransaction(
 				foundSpanType = httpSpan
 				httpServerName = stringval
 			case semconv.AttributeHTTPClientIP:
-				event.Client.IP = net.ParseIP(stringval)
+				event.Client.IP = netip.MustParseAddr(stringval)
```

Comment:
We shouldn't be using the `Must` function here since it'll cause APM Server to panic. We don't seem to have a panic / recover interceptor for gRPC, so it's likely to cause APM Server to crash. Instead, we should use [`netip.ParseAddr`](https://pkg.go.dev/net/netip@go1.18.5#ParseAddr) an only assign the value on `err == nil`
```suggestion
				if ip, err := netip.ParseAddr(stringval); err == nil {
					event.Client.IP = ip
				}
```

Reply:
Thanks for spotting this!